docs/sphinx/_toc.yml.in

```diff
@@ -4,14 +4,17 @@ root: index
 subtrees:
   - caption: Install
     entries:
-    - file: installation    
+    - file: installation
   - caption: API reference
     entries:
     - file: datatypes
     - file: api-reference
     - file: ext-reference
     - file: ext-ops
     - file: logging-heuristics
+  - caption: How To
```

Comment:
```suggestion
  - caption: How to
```

Reply:
Sorry, that was my mistake when I made the suggestion. Should be "How to" and not "How To".